code/game/gamemodes/game_mode.dm

```diff
@@ -112,6 +112,10 @@ GLOBAL_VAR_INIT(cas_tracking_id_increment, 0) //this var used to assign unique t
 	log_game("Server IP: [world.internet_address]:[world.port]")
 	return TRUE
 
+/datum/game_mode/proc/adjust_ammo_values()
+	for(var/ammo in GLOB.ammo_list)
+		GLOB.ammo_list[ammo].setup_hvh_values()
```

Comment:
shouldn't this just immediately return, and it's overridden for /datum/game_mode/faction_clash to do the same thing, so we don't have to do `if(MODE_HAS_FLAG(MODE_FACTION_CLASH))` for every piece of ammo? 

alternatively, the MODE_HAS_FLAG should be here, not in the ammo